shell/protoc.sh

```diff
@@ -42,16 +57,20 @@ docker exec "$CONTAINER_ID" sh -c "groupadd -f --gid $gid localuser && useradd -
 # Create the language specific clients
 info_sub "go"
 docker exec --user localuser "$CONTAINER_ID" entrypoint.sh -f './*.proto' -l go \
+  $(for import in $(get_list "go-protoc-imports"); do echo "-i /mod/$(get_import_basename "$import")"; done) \
+  $(if has_feature "validation"; then echo "--with-validator --validator-source-relative"; fi) \
```

Comment:
What's this validation functionality?

Reply:
I can break this into a separate change, but this basically enables using https://github.com/envoyproxy/protoc-gen-validate with our stuff. It's the other piece that settings service needs to get fully unblocked on managing its schema. Any opposition to bringing it in, in general?

Reply:
Can we have a discussion on this in #go before we make this choice?  I don't have a strong opinion on it itself but if we do this we are effectively encouraging folks to use `protoc-gen-validate` as the way to validate constraints and seems like we should see if anyone else has any opinions on this.

Reply:
Ah, no opposition just wasn't super sure what it was!

Reply:
I'll go ahead and pull this out into it's own PR and get a discussion going in #go